src/screens/scanning/ManualVoucherScreen.tsx

```diff
@@ -17,14 +21,12 @@ import {
 } from './styles';
 import InputField from '../../components/InputField/InputField';
 import StandardLogo from '../../components/common/StandardLogo';
-import {
-  validateSerialNumberInput,
-  validateVoucherAmount,
-} from '../../utils/validationUtils';
+import { validateSerialNumberInput } from '../../utils/validationUtils';
 
 function ManualVoucherScreen() {
   const [transactionID, setID] = useState<string>('');
-  const [voucherAmount, setVoucherAmount] = useState<string>('');
+  const [voucherAmount, setVoucherAmount] = useState<number | null>(0);
```

Comment:
I'm noticing that if you fully delete everything in this field, it clears the field rather than showing `0.00` like I would expect, and I suspect it's cleared when `voucherAmount` is `null`.

To make it always show `0.0`, you could probably make this `voucherAmount` only `number`, and in `onChangeVoucherAmount`, if `value` is null, `setVoucherAmount(0)`. You could do this using something like
```
setVoucherAmount(value ?? 0)
```
to make it fallback to 0.